.github/workflows/test-vurnabilities-data.yml

```
@@ -0,0 +1,124 @@
+        if: always()
+        env:
+          DB_HOST: ${{ secrets.CYPRESS_DB_HOST }}
+          DB_NAME: ${{ secrets.CYPRESS_DB_NAME }}
+          DB_USER: ${{ secrets.CYPRESS_DB_USER }}
+          DB_PWD: ${{ secrets.CYPRESS_DB_PWD }}
+          GITHUB_PR_ID: ${{ github.event.pull_request.number }}
+          GITHUB_PR_LINK: ${{ github.event.pull_request.html_url }}
+          GITHUB_RUN_ID: ${{ github.run_id }}
+        run: |
+          chmod +x scripts/scout_vulnerabilities_data.sh
+          ./scripts/scout_vulnerabilities_data.sh "${{ inputs.image_name }}"
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)